exporter/elasticsearchexporter/logs_exporter_test.go

```diff
@@ -38,6 +38,13 @@ func TestExporter_New(t *testing.T) {
 		require.Nil(t, err)
 		require.NotNil(t, exporter)
 	}
+	successWithInternalModel := func(expectedModel *encodeModel) validate {
+		return func(t *testing.T, exporter *elasticsearchLogsExporter, err error) {
+			require.Nil(t, err)
+			require.NotNil(t, exporter)
```

Comment:
This line isn't needed :) 

It would also be best to switch to `assert` here since I don't believe these values are consumed later making the test flaky

Reply:
Hi @MovieStoreGuy, sorry, just to clarify, which lines aren't needed? 43-44? Or are you looking for an anonymous function?

Reply:
Line 44, you can leave it as `assert.EqualValues`

Reply:
@MovieStoreGuy fixed, thanks